src/legacy/components/Dashboard.tsx

```diff
@@ -1,18 +1,8 @@
 /* eslint-disable */
 
-import React, { Component } from 'react'
```

Comment:
Please don't change the legacy code

Reply:
I'm doing the route organization in the next PR (WIP, portfolio pages), and this is not a quick fix because current new routes are dedicated to onboarding flow, so need changes
I will keep you posted with the progress

Reply:
see the above comment